app/assets/stylesheets/partials/_overlay.scss

```diff
@@ -0,0 +1,48 @@
+.overlay {
+  display: grid;
+  grid-template-columns: 16.3125rem 1fr 5rem 11.3125rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+  background-color: rgba(63, 60, 60, 0.705);
+  overflow: hidden;
+
+  @include media(small){
```

Comment:
Minor styling request to include a space after @include media(small). See https://github.com/michaelherold/codeclimate-sass-lint/blob/master/config/contents/space-before-brace.md